src/globus_sdk/services/flows/client.py

```diff
@@ -46,14 +46,14 @@ def create_flow(
         additional_fields: dict[str, t.Any] | None = None,
     ) -> GlobusHTTPResponse:
         """
-        Create a Flow
+        Create a flow
```

Comment:
😬

Reply:
We need a style-guide for this. We started using emphasis (boldface, IIRC?) in some of the docs for `flow`, `run`, `timer`, and other proper nouns which are easily confused with service names.

Reply:
Yeah, I think the way I'd phrase it is roughly:

> Entities (e.g., flow, run, action, action provider) are lowercase and first appearance is set off as a "term" in whatever paradigm a given document uses; for the docs site, we use bold

Is there such a paradigm—or related prior-art—in the SDK docs? If so, happy to make consistent!